rules/kubecf/release_chart.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This is a helper script which combines a helm chart tarball and an image list
+# into a new tarball.
+
+if [ ! -e "$1" ]; then
```

Comment:
If you declare some vars at the beginning of the script mapping the arguments, it's easier to consume the script itself and to read the script.

```suggestion
# into a new tarball.

HELM_CHART="$1"
IMAGE_LIST_FILE="$2"
OUTPUT="$3"
JQ_BINARY="$4"

if [ ! -e "${HELM_CHART}" ]; then
```

Reply:
Done in https://github.com/cloudfoundry-incubator/kubecf/pull/634/commits/42b322103f8bd3dc45962383f317702e9d09d476.